system/protocols/mumble/protocol.py

```diff
@@ -19,20 +7,34 @@
 from twisted.internet import reactor, ssl, task
 
 from system.commands.manager import CommandManager
+
 from system.enums import CommandState
+
 from system.events.manager import EventManager
 from system.events import general as general_events
 from system.events import mumble as mumble_events
+
+from system.logging.logger import getLogger
+
 from system.protocols.generic.protocol import SingleChannelProtocol
+
 from system.protocols.mumble import Mumble_pb2
 from system.protocols.mumble.user import User
 from system.protocols.mumble.channel import Channel
+from system.protocols.mumble.acl import Perms
+from system.protocols.mumble.structs import Version
+
+from system.translations import Translations
 
 from utils.html import html_to_text
-from system.logging.logger import getLogger
+from utils.protobuf import decode_varint
 from utils.switch import Switch
+__author__ = 'Gareth Coles'
 
-from system.translations import Translations
+# This is a modified version of Chaosteil's open-domain Mumble library.
+# The original code can be found on GitHub, at the following link..
+# https://github.com/Chaosteil/rdiablo-mumble-bot/blob/master/
```

Comment:
This URL doesn't work, remove the `blob/master/` part, link directly to the `mumble_protocol.py` file, or just remove the comment.